pkg/controller/import-controller.go

```diff
@@ -782,6 +766,16 @@ func (r *ImportReconciler) createScratchPvcForPod(pvc *corev1.PersistentVolumeCl
 		anno[AnnBoundConditionMessage] = "Creating scratch space"
 		anno[AnnBoundConditionReason] = creatingScratch
 	} else {
+		if scratchPvc.DeletionTimestamp != nil {
```

Comment:
this can only happen if we reuse the pod name or scratch name, is that the case?

Reply:
yes, and this is exactly what happens with a warm migration, well before #2146 anyway.